pkgs/applications/video/obs-studio/plugins/default.nix

```diff
@@ -31,4 +31,6 @@
   obs-websocket = throw "obs-websocket has been removed: Functionality has been integrated into obs-studio itself.";
 
   wlrobs = callPackage ./wlrobs.nix { };
+
+  input-overlay = qt6Packages.callPackage ./input-overlay.nix { };
```

Comment:
Is/will there be an issue when using another name than pname here with package search or sth?

I initially tried to import it with "obs-input-overlay" which won't work as it is exported as "input-overlay" this made me think something wasn't right.

Reply:
The canonical name seems to be `input-overlay` which is what we should use IMHO. The pname is only the name in the nix store which is not related the attribute name in any way. It could be foobar. :stuck_out_tongue: